docs/resources/sensitive_data_scanner_rule.md

```diff
@@ -67,6 +67,7 @@ resource "datadog_sensitive_data_scanner_rule" "mylibraryrule" {
 
 - `description` (String) Description of the rule.
 - `excluded_namespaces` (List of String) Attributes excluded from the scan. If namespaces is provided, it has to be a sub-path of the namespaces array.
+- `included_keyword_configuration` (Block List, Max: 1) Object defining a set of keywords and a number of characters that help reduce noise. You can provide a list of keywords you would like to check within a defined proximity of the matching pattern. If any of the keywords are found within the proximity check then the match is kept. If none are found, the match is discarded. (see [below for nested schema](#nestedblock--included_keyword_configuration))
```

Comment:
Could we update the example of usage as well?

Reply:
[Add new attribute in examples](https://github.com/DataDog/terraform-provider-datadog/pull/2238/commits/f03ee1d56521aa87a86c21d37da97bab1dacf3ab)